src/crimson/os/seastore/omap_manager/btree/btree_omap_manager.h

```diff
@@ -27,7 +27,8 @@ class BtreeOMapManager : public OMapManager {
 
   omap_context_t get_omap_context(
     Transaction &t, laddr_t addr_min) {
-    return omap_context_t{tm, t, addr_min};
+    uint32_t deviation = (((uint32_t)std::rand() & OMAP_LADDR_SPACE_MASK) >> 12) << 12;
+    return omap_context_t{tm, t, addr_min + deviation};
```

Comment:
Is the idea here to avoid having the omap addresses for a particular onode cluster around the hint?

Reply:
yes:-)